ubuntu_install_requirements.sh

```diff
@@ -18,36 +18,39 @@ sudo apt -y update
 # Install required packages
 
 sudo apt -y install \
+  python3-pip \
+  python-pip \
+  python-setuptools \
   zlib1g-dev \
   libssl1.0-dev \
   wget
 
-# Install pyenv
-
-if [[  $(cat ~/.bashrc) != *PYENV_ROOT* ]]; then
-  if ! [ -d "$HOME/.pyenv" ] ; then
-     git clone git://github.com/yyuu/pyenv.git ~/.pyenv
-  fi
-  # shellcheck disable=SC2016
-  # shellcheck disable=SC2129
-  echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
-  # shellcheck disable=SC2016
-  echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
-  # shellcheck disable=SC2016
-  echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.bashrc
-fi
+# # Install pyenv
+
+# if [[  $(cat ~/.bashrc) != *PYENV_ROOT* ]]; then
+#   if ! [ -d "$HOME/.pyenv" ] ; then
+#      git clone git://github.com/yyuu/pyenv.git ~/.pyenv
+#   fi
+#   # shellcheck disable=SC2016
+#   # shellcheck disable=SC2129
+#   echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+#   # shellcheck disable=SC2016
+#   echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+#   # shellcheck disable=SC2016
+#   echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.bashrc
+# fi
```

Comment:
let's completely remove it for now, to have it cleaner

Reply:
Done